pkg/internal/containerd_conf.toml

```diff
@@ -64,8 +64,8 @@ version = 2
     unset_seccomp_profile = ""
 
     [plugins."io.containerd.grpc.v1.cri".cni]
-      bin_dir = "C:\\k\\cni"
-      conf_dir = "C:\\k\\cni\\config"
+      bin_dir = "C:\\Windows\\Temp\\cni"
```

Comment:
yes I tried to explain it in the commit message for this commit.
Basically the code for copying theses files to C:\\k directory was part of configure CNI test file and depended on information from some structs which are now removed from the dev code.
I could refactor and add back those methods if required but I felt we could directly set the path to the location they are created in initially as this is only going to be used for our test cases.

Reply:
Will this need to be changed on the [WMCO side](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/internal/containerd_conf.toml#L67-L68) as well?

Reply:
No it wouldn't, WMCO expects them into be that location as it copies them to that loaction.
This change is only limited to WMCB tests which mimic the same WMCO behavior so that networking is setup and node becomes ready.